src/main/resources/site/site.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<site xmlns="urn:enonic:xp:model:1.0">
```

Comment:
Lets follow the backend recommendation and follow the xml model. 
X-data needs to be before form 🤷 And we can add the xmlns to it, since it might help some new developers.

Reply:
Okay